src/Example.jl

```diff
@@ -1,7 +1,18 @@
 module Example
 export hello, domath
 
+"""
+    hello(who::String)
+
+Returns "Hello, `who`"
```

Comment:
The manual recommends using the imperative form instead:
```suggestion
Return "Hello, `who`".
```
Same below.

Reply:
This wasn't actually fixed.

Reply:
Whoops, sorry I was dealing with something else and apparently didn't actually read the comment.